RSFormView/Classes/Views/TextField/TextFieldView.swift

```diff
@@ -274,6 +274,16 @@ extension TextFieldView: UITextFieldDelegate {
     return true
   }
   
+  func textFieldShouldBeginEditing(_ textField: UITextField) -> Bool {
+    guard let data = fieldData,
+        data.fieldType == .picker else { return true }
+    guard let fieldOptions = fieldData?.options else { return true }
```

Comment:
Please join these two guard conditions.

Reply:
Apologies! Silly oversight, should be fixed.